spec/routing/routes_spec.rb

```diff
@@ -0,0 +1,1602 @@
+      end
+
+      #it 'routes POST populate_repo_browser properly' do
+      #       expect(post: sub_path + '/populate_repo_browser').to route_to(
+      #         controller: sub_ctrl,
+      #         action: 'populate_repo_browser',
+      #         assignment_id: assignment.id.to_s,
+      #         locale: 'en')
+      #end
+
+      it 'routes POST update_converted_pdfs properly' do
+        expect(post: sub_path + '/update_converted_pdfs').to route_to(
+          controller: sub_ctrl,
+          action: 'update_converted_pdfs',
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+
+      it 'routes GET updated_files properly' do
+        expect(get: sub_path + '/updated_files').to route_to(
+          controller: sub_ctrl,
+          action: 'updated_files',
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+
+      it 'routes GET replace_files properly' do
+        expect(get: sub_path + '/replace_files').to route_to(
+          controller: sub_ctrl,
+          action: 'replace_files',
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+
+      it 'routes GET delete_files properly' do
+        expect(get: sub_path + '/delete_files').to route_to(
+          controller: sub_ctrl,
+          action: 'delete_files',
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+
+      it 'routes POST update_files properly' do
+        expect(post: sub_path + '/update_files').to route_to(
+          controller: sub_ctrl,
+          action: 'update_files',
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+
+      it 'routes POST server_time properly' do
+        expect(post: sub_path + '/server_time').to route_to(
+          controller: sub_ctrl,
+          action: 'server_time',
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+
+      it 'routes GET download properly' do
+        expect(get: sub_path + '/download').to route_to(
+          controller: sub_ctrl,
+          action: 'download',
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+    end
+    
+    context 'member' do
+      
+      it 'routes GET collect_and_begin_grading properly' do
+        expect(get: sub_path + '/' + submission.id.to_s + '/collect_and_begin_grading').to route_to(
+          controller: sub_ctrl,
+          action: 'collect_and_begin_grading',
+          id: submission.id.to_s,
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+      end
+
+      it 'routes POST manually_collect_and_begin_grading properly' do
+        expect(post: sub_path + '/' + submission.id.to_s + '/manually_collect_and_begin_grading').to route_to(
+          controller: sub_ctrl,
+          action: 'manually_collect_and_begin_grading',
+          id: submission.id.to_s,
+          assignment_id: assignment.id.to_s,
+          locale: 'en')
+       end
```

Comment:
`end` at 730, 7 is not aligned with `it 'routes POST manually_collect_and_begin_grading properly' do` at 723, 6